views/partials/sign-up-banners/breach-detail-sign-up-banner.hbs

```diff
@@ -1,7 +1,7 @@
 
       <div class="sign-up-wrapper breach-detail-sign-up drop-shadow fxa-signup-gradient">
         <div class="mw-8 sign-up-banner">
-          <div class="sign-up-headline-list-wrapper flx flx-col">
+          <div class="flx flx-col">
```

Comment:
Should the `sign-up-headline-list-wrapper` class be removed from CSS to, or is it still in use on the Breach Details signup?